tests/rl/smoke_test.sh

```
@@ -0,0 +1,277 @@
+        kill "$pid" 2>/dev/null || true
+    done
+}
+trap cleanup EXIT INT TERM
+
+MODEL="${1:-Qwen/Qwen3-0.6B}"
+HTTP_PORT="${DYN_HTTP_PORT:-8000}"
+NATS_PORT="${NATS_PORT:-4222}"
+# prime_rl source must be on PYTHONPATH so the spawned vLLM worker subprocess
+# can import prime_rl.inference.vllm.worker.filesystem.FileSystemWeightUpdateWorker.
+PRIME_RL_SRC="${PRIME_RL_SRC:-/home/biswaranjanp/dev/rl/prime-rl/src}"
+PYTHON="${SMOKE_PYTHON:-python}"
```

Shellcheck (0.11.0)

[warning] PYTHON appears unused. Verify use (or export if used externally).

(SC2034)